node/network/gossip-support/src/tests.rs

```diff
@@ -254,6 +275,17 @@ fn issues_a_connection_request_on_new_session() {
 
 		test_neighbors(overseer).await;
 
+		assert_matches!(
+			overseer_recv(overseer).await,
+			AllMessages::RuntimeApi(RuntimeApiMessage::Request(
+				relay_parent,
+				RuntimeApiRequest::SessionInfo(1, sender),
+			)) => {
+				assert_eq!(relay_parent, hash);
+				sender.send(Ok(Some(SESSION_INFO.clone()))).unwrap();
```

Comment:
alternatively, you can simply send `None` here, doesn't seem to be used anyway

Reply:
done